README.md

```diff
@@ -162,6 +162,13 @@ sct.add_transaction(
   # String, 8 or 11 char
   bic:                    'PBNKDEFF370',
 
+  # OPTIONAL: IID of the bank (Institution Identification)
+  # if IID is used the code is mandatory
+  # String, 8 or 11 char
+  iid:                    '700',
+  # OPTIONAL: Clearing code of the country
+  code:                    'CHBCC',
```

Comment:
Is it clear that "code" stands for "clearing code of the country" throughout the whole app?